packages/@react-spectrum/s2/src/StatusLight.tsx

```diff
@@ -40,7 +40,7 @@ export interface StatusLightProps extends StatusLightStyleProps, DOMProps, AriaL
   /**
    * The content to display as the label.
    */
-  children?: ReactNode,
+  children: ReactNode,
```

Comment:
I don't recall if this one can just be the standalone without a visible label, I think Spectrum discourages it? so probably ok to require this one